west.yml

```diff
@@ -98,7 +98,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 478f03a71256aeb2890e685ebf5a1dc6832a6b5f
+      revision: pull/49/head
```

Comment:
Please move this commit to #26785 and close this PR.

Reply:
Done, I am closing this PR.